content/proposals/ctv.md

```diff
@@ -36,6 +36,13 @@ coinjoin transactions in new ways that might simplify construction or reduce fee
 have mentioned that the new opcode could possibly be used to allow users to trustlessly pool their
 funds together into a single UTXO in a way that would increase privacy.
 
+The original proposal also emphasized other contracts and covenants that could be created using the new opcode,
+such as the ability to create channel factories, vaults, coinjoin transactions in new ways that might simplify
+construction or reduce fees, and described how the new opcode could possibly be used to allow users to trustlessly
+pool their funds together into a single UTXO in a way that would increase privacy. These other concepts have been
+described in further detail in subsequent works by a variety of authors.
```

Comment:
Could you add links to some of these concepts? I believe you're referring to some of your utxos.org content, which would be helpful context for new readers.

Reply:
I think that makes sense for the optech page as a separate patch, but the covenants.info does have more resources for covenant applications throughout the site.

Reply:
Good point. The optech link is certainly more comprehensive.